packages/cli/generators/relation/templates/controller-relation-template-has-many.ts.ejs

```diff
@@ -58,7 +58,7 @@ export class <%= controllerClassName %> {
     @requestBody({
       content: {
         'application/json': {
-          schema: getModelSchemaRef(<%= targetModelClassName %>, {exclude: ['<%= targetModelPrimaryKey %>']}),
+          schema: getModelSchemaRef(<%= targetModelClassName %>, {exclude: ['<%= targetModelPrimaryKey %>'], optional: ['<%= foreignKeyName %>']}),
```

Comment:
Would it be possible to break this into new lines? e.g.:
```
schema: getModelSchemaRef(<%= targetModelClassName %>, {
  exclude: ['<%= targetModelPrimaryKey %>'],
  optional: ['<%= foreignKeyName %>'],
}),
```

Reply:
You can try to generate the code with your PR and format it with `npm run prettier:fix`. Use the formatted code as reference to improve the template.

Reply:
fixed